osgar/tools/lidarview.py

```diff
@@ -313,6 +311,16 @@ def _step(self, direction):
                 self.scan2 = deserialize(data)
             elif stream_id == self.camera_id:
                 self.image = get_image(deserialize(data))
+                current = self.current
+                while current + direction >= 0 and current + direction < len(self.log):
```

Comment:
I would add here description of why you are doing this (could be copy & paste from PR description(